backend-cm/package.json

```diff
@@ -32,7 +32,9 @@
     "test:cov": "jest --coverage --detectOpenHandles --forceExit",
     "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
     "test:e2e": "jest --config ./test/jest-e2e.json",
-    "import:parks": "nest start --entryFile=./src/cli.js -- import -j parks"
+    "import:all": "nest start --entryFile=./src/cli.js -- import -j parks;bclaws",
```

Comment:
The intention here is to use the same import job and pass a delimited set of imports to run